modules/react/table/lib/Table.tsx

```diff
@@ -8,18 +8,67 @@ import {TableHeader} from './TableHeader';
 import {TableCell} from './TableCell';
 import {TableFooter} from './TableFooter';
 import {TableCaption} from './TableCaption';
-import {createStencil, px2rem} from '@workday/canvas-kit-styling';
+import {calc, createStencil, px2rem} from '@workday/canvas-kit-styling';
 import {system} from '@workday/canvas-tokens-web';
 
-const tableStencil = createStencil({
+export interface TableProps extends GridProps {
+  /**
+   * Variant has an option for `simple` which will have the Table act as a HTML Table without Grid styles.
+   */
+  variant?: 'html';
```

Comment:
Maybe this is `table | grid` with the default being gird? Not sure html conveys that it behaves like a default html table element.

Reply:
Yup. Going to be addressing this.